flink-runtime-web/web-dashboard/src/app/pages/overview/statistic/overview-statistic.component.html

```diff
@@ -58,4 +66,42 @@
       </ng-container>
     </nz-card>
   </div>
+  <div nz-col [nzXs]="24" [nzSm]="24" [nzMd]="24" [nzLg]="24" [nzXl]="24" class="col">
+    <nz-card [nzBordered]="false" nzTitle="Task Manager Overview" [nzLoading]="!listOfTaskManager">
+      <ng-container *ngFor="let manager of listOfTaskManager;">
```

Comment:
```suggestion
      <ng-container *ngFor="let manager of listOfTaskManager;trackBy:trackByFunc">
```
should add `trackBy` function here to avoid the dom rerender after refresh

the `trackByFunc` could be

```typescript
  trackByFunc(_: number, item: TaskmanagersItemInterface) {
    return item.id;
  }
```